terraform/deploy/cognito/modules.tf

```diff
@@ -0,0 +1,13 @@
+module "cognito" {
+  source = "../../modules/cognito"
+
+  common_tags = local.common_tags
+
+  clients = [
+    "jupyterhub",
+  ]
+
+  root_dns_names = values(local.root_dns_name)
+
+  domain = local.management_account[local.environment] == "management" ? "dataworks" : "dataworks-dev"
```

Comment:
I wonder if this is worth interpolating inside of `locals` instead of here and doing a simple reference?